exercices/core.clj

```diff
@@ -0,0 +1,65 @@
+(ns cipher.core
+  (:require [clojure.string :as str])
+  (import [java.lang Character]))
+
+(defn foo
+  "I don't do a whole lot."
+  [x]
+  (println x "Hello, World!"))
+
+
+(defn to-int [valor]
+  (let [asca (int \a)]
+    (- (int valor) asca)))
+
+(defn to-char [valor]
+  (let [asca (int \a)]
+    (char (+ valor asca))))
+
+(defn shift [letra valor]
+  (to-char (mod (+ (to-int letra) valor) 26)))
+; (-> (to-int letra)
+;     (+ n)
+;     (mod 26)
+;     (to-char))
+
+(defn caesar-encrypt [palavra valor]
+  (apply str (mapv #(shift % valor) palavra)))
+
+
+(defn caesar-decrypt [palavra valor]
+  (apply str (mapv #(shift % (- valor)) palavra)))
+
+
+(defn get-letters [palavra]
+   (->> (filterv #(Character/isLetter %) palavra)
+        (apply str)
+        (str/lower-case)))
```

Comment:
Minor, for cases like that you can call the function without parenthesis, replace `(str/lower-case)` -> `str/lower-case`